src/include/access/tableam.h

```diff
@@ -53,6 +53,13 @@ struct ValidateIndexState;
 */
 #define SCAN_SUPPORT_COLUMN_ORIENTED_SCAN (1 << 0)  /* support column-oriented scanning*/
 
+/*
+ * Flags represented the supported features of scan.
+ *
+ * Reserve first 8 bits for public.
+ */
+#define SCAN_SUPPORT_RUNTIME_FILTER (1 << 0) /* support runtime filter scan */
```

Comment:
The flag conflicts with `SCAN_SUPPORT_COLUMN_ORIENTED_SCAN`

Reply:
change to `#define SCAN_SUPPORT_RUNTIME_FILTER       (1 << 1)  /* support runtime filter scan */`